console/src/view/mod.rs

```diff
@@ -109,21 +121,35 @@ impl View {
         &mut self,
         frame: &mut tui::terminal::Frame<B>,
         area: layout::Rect,
-        tasks: &mut crate::tasks::State,
+        state: &mut State,
     ) {
+        let styles = &self.styles;
         match self.state {
             ViewState::TasksList => {
-                self.list.render(&self.styles, frame, area, tasks);
+                self.tasks_list.render(styles, frame, area, state);
             }
             ViewState::TaskInstance(ref mut view) => {
-                let now = tasks
+                let now = state
                     .last_updated_at()
                     .expect("task view implies we've received an update");
                 view.render(&self.styles, frame, area, now);
             }
         }
 
-        tasks.retain_active();
+        state.retain_active();
+    }
+
+    pub(crate) fn current_view(&self) -> &ViewState {
+        &self.state
+    }
+}
+
+impl ViewState {
+    pub(crate) fn is_tasks_view(&self) -> bool {
+        if let Self::TasksList = self {
+            return true;
+        }
+        false
```

Comment:
nit: we could simplify this to just be
```suggestion
        matches!(self, Self::TasksList)
```